mainnet/13-wysiwyg-finance-transfer/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "voting": "0x0e835020497b2cd716369f8fc713fb7bd0a22dbf",
```

Comment:
I didn't see us using this anywhere, but I presume we would have to change this for the real voting app's address?

Reply:
It is being used in the `create-disputed-vote` script, it is the voting app we are using to create the votes :)